### PR TITLE
Feat: expire expired bids

### PR DESCRIPTION
**Pull request by nachomazzara**

### CODE REVIEW

File: shared/listing.js [hunk 1]
```diff
@@ -26,7 +27,7 @@ export const LISTING_SORT_BY = Object.freeze({
 
 export const DEFAULT_DAY_INTERVAL = 31
 export const MINIMUM_DAY_INTERVAL = 1
-export const MAXIMUM_BID_DAY_INTERVAL = 6 * 31 // six month
+export const MAXIMUM_BID_DAY_INTERVAL = 182 // six month
```
Comment (nicosantangelo) [inline shared/listing.js]:
```suggestion
export const MAXIMUM_BID_DAY_INTERVAL = 182 // six months
```

😛 

Also why this change?

File: src/Listing/Listing.js [hunk 2]
```diff
@@ -46,10 +48,22 @@ export class Listing {
...
     return this.Model.find({ owner })
   }
 
+  async updateExpired() {
+    return db.query(SQL`
+      UPDATE ${raw(this.tableName)}
+        SET updated_at = NOW(),
+            status = ${LISTING_STATUS.expired}
+      WHERE ${ListingQueries.isInactive()}
+        AND ${ListingQueries.hasStatus([
```
Comment (nachomazzara) [inline src/Listing/Listing.js]:
```suggestion
        AND ${ListingQueries.hasStatuses([
```
